package.json

```diff
@@ -29,28 +29,28 @@
   },
   "homepage": "https://github.com/massfords/ts-rsql-query#readme",
   "devDependencies": {
-    "@types/common-tags": "^1.8.1",
-    "@types/jest": "^28.1.3",
-    "@types/uuid": "^8.3.4",
-    "@typescript-eslint/eslint-plugin": "^5.30.0",
-    "@typescript-eslint/parser": "^5.30.0",
-    "eslint": "^8.18.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-jest": "^26.5.3",
-    "eslint-plugin-prettier": "^4.1.0",
-    "jest": "^28.1.1",
-    "pg-promise": "^10.11.1",
-    "prettier": "^2.7.1",
+    "@types/common-tags": "^1.8.4",
+    "@types/jest": "^29.5.14",
+    "@types/uuid": "^10.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.11.0",
+    "@typescript-eslint/parser": "^8.11.0",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^8.10.0",
+    "eslint-plugin-jest": "^28.8.3",
+    "eslint-plugin-prettier": "^5.2.1",
+    "jest": "^29.7.0",
+    "pg-promise": "^11.10.1",
+    "prettier": "^3.3.3",
     "testcontainers": "^9.1.3",
```

Comment:
deliberately avoided bump to testcontainers. Need to look at what changed with respect to the Postgres support. I think their exports changed but don't have time now to look at it

Reply:
Yeah, guess that should not be a big issue.